docs/parser/comparator/runner-php-sync.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+$rss[ 'start' ] = mem();
+
+ini_set( 'max_execution_time', 120 );
+ini_set( 'memory_limit', '512M' );
+
+$count = $_GET['count'] ?: 1;
+$document = file_get_contents( 'php://input' );
+
+$rss[ 'beforeParserInit' ] = mem();
+require_once '/parser.php';
+$my_parser = new MyParser();
+$rss[ 'afterParserInit' ] = mem();
+
+$tic = microtime( true );
+$parse = $my_parser->parse( $document );
+$sentinel = 1;
+for ( $i = 1; $i < $count; $i++ ) {
+    $my_parser->parse( $document );
+    $sentinel += $i;
+}
+$toc = microtime( true );
+$rss[ 'end' ] = mem();
+
+$runtime = ( $toc - $tic ) * 1e6;
+
+header( 'Access-Control-Allow-Origin: *' );
+echo json_encode( [
+    'parse' => $parse,
+    'µs' => $runtime,
```

Comment:
µs or us? cf https://github.com/WordPress/gutenberg/blob/c41a38fe1914de169aeaa25f62efbcd7e63cbcd3/docs/parser/comparator/README.md

Reply:
good catch! I changed it because Visual Studio Code was inserting an unexpected character when I typed µ.